src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ApprenticeshipDetailsRequestToViewModelMapperTests.cs

```diff
@@ -90,7 +91,7 @@ public void SetUp()
             _mockEncodingService.Setup(t => t.Decode(It.IsAny<string>(), It.IsAny<EncodingType>()))
                 .Returns((string value, EncodingType encodingType) => long.Parse(Regex.Replace(value, "[A-Za-z ]", "")));
 
-            _mapper = new ApprenticeshipDetailsRequestToViewModelMapper(_mockCommitmentsApiClient.Object, _mockEncodingService.Object, Mock.Of<ILogger<ApprenticeshipDetailsRequestToViewModelMapper>>());
+            _mapper = new ApprenticeshipDetailsRequestToViewModelMapper(_mockCommitmentsApiClient.Object, _mockEncodingService.Object, Mock.Of<ILogger<ApprenticeshipDetailsRequestToViewModelMapper>>(), Mock.Of<IAuthorizationService>());
```

Comment:
Should we add a test here to ensure ensure we map  ShowApprenticeConfirmationColumn and ConfirmationStatus correctly